public/components/trace_analytics/components/common/filters/filters.tsx

```diff
@@ -318,14 +224,117 @@ export function Filters(props: FiltersOwnProps) {
   const filterComponents = useMemo(() => renderFilters(), [props.filters]);
 
   return (
-    <EuiFlexGroup gutterSize="xs" alignItems="center" responsive={false} style={{ minHeight: 32 }}>
-      <EuiFlexItem grow={false}>
-        <GlobalFilterButton />
-      </EuiFlexItem>
+    <EuiFlexGroup
+      gutterSize="xs"
+      alignItems="center"
+      responsive={false}
+      style={{ minHeight: 32, padding: '0 16px' }}
```

Comment:
Why do we need custom styling here? We should Ideally stick to EUI/OUI component styling.

Reply:
<img width="352" alt="Screenshot 2024-08-22 at 12 49 09 PM" src="https://github.com/user-attachments/assets/a401faeb-2b9f-452b-8dfa-befca5388d06">

To place the add filter in the correct allignment. I didn't see any prop options that would allow me to add the padding without nesting it in a page which creates a ton of space between it and the groups.
<img width="296" alt="Screenshot 2024-08-22 at 1 02 43 PM" src="https://github.com/user-attachments/assets/c632de73-f610-4237-aa45-5c150daffcd9">

Reply:
Follow up question how is mode picker getting the page? 
Also, would it make sense to keep the filter and mode picker (with search bar) inside the EuiPage itself?

Reply:
Yes, adjusted it